core/indexer/process/serialize_test.go

```diff
@@ -0,0 +1,75 @@
+package process
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/ElrondNetwork/elrond-go/core/indexer/types"
+	"github.com/stretchr/testify/require"
+)
+
+func TestPrepareBufferMiniblocks(t *testing.T) {
+	var buff bytes.Buffer
+
+	meta := []byte("test1")
+	serializedData := []byte("test2")
+
+	buff = prepareBufferMiniblocks(buff, meta, serializedData)
+
+	var expectedBuff bytes.Buffer
+	serializedData = append(serializedData, "\n"...)
+	expectedBuff.Grow(len(meta) + len(serializedData))
+	_, _ = expectedBuff.Write(meta)
+	_, _ = expectedBuff.Write(serializedData)
+
+	require.Equal(t, expectedBuff, buff)
+}
+
+func TestSerializeScResults(t *testing.T) {
+	t.Parallel()
+
+	scResult1 := &types.ScResult{
+		Hash:     "hash1",
+		Nonce:    1,
+		GasPrice: 10,
+		GasLimit: 50,
+	}
+	scResult2 := &types.ScResult{
+		Hash:     "hash2",
+		Nonce:    2,
+		GasPrice: 10,
+		GasLimit: 50,
+	}
+	scrs := []*types.ScResult{scResult1, scResult2}
+
+	res, err := serializeScResults(scrs)
+	require.Nil(t, err)
+	require.Equal(t, 1, len(res))
+
+	expectedRes := "{ \"index\" : { \"_id\" : \"hash1\" } }\n{\"nonce\":1,\"gasLimit\":50,\"gasPrice\":10,\"value\":\"\",\"sender\":\"\",\"receiver\":\"\",\"prevTxHash\":\"\",\"originalTxHash\":\"\",\"callType\":\"\",\"timestamp\":0}\n{ \"index\" : { \"_id\" : \"hash2\" } }\n{\"nonce\":2,\"gasLimit\":50,\"gasPrice\":10,\"value\":\"\",\"sender\":\"\",\"receiver\":\"\",\"prevTxHash\":\"\",\"originalTxHash\":\"\",\"callType\":\"\",\"timestamp\":0}\n"
```

Comment:
use ` instead of " to avoid the quotes escaping

Reply:
done.